executor/common.h

```diff
@@ -111,6 +117,7 @@ static void install_segv_handler(void)
 	sigaction(SIGBUS, &sa, NULL);
 }
 
+#if !GOOS_windows
```

Comment:
There is already `#if !GOOS_fuchsia && !GOOS_windows` on top of this whole section related to signals. Is this needed?

Reply:
I'm sorry about this. I did not notice that.